pom.xml

```diff
@@ -49,7 +49,7 @@
         <immutables.version>2.7.1</immutables.version>
         <jackson.databind.nullable.version>0.2.0</jackson.databind.nullable.version>
         <jackson.databind.version>2.9.9.1</jackson.databind.version>
-        <jackson.version>2.9.9</jackson.version>
+        <jackson.version>2.10.0.pr1</jackson.version>
```

Comment:
I think we should close this PR and wait until a new 2.9.x version or 2.10.0 is released.